src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -67,6 +67,22 @@ public boolean hasPerson(Person person) {
         return persons.contains(person);
     }
 
+    /**
+     * Returns true if a person with the same phone number as {@code person} exists in the address book.
+     */
+    public boolean hasPhone(Person person) {
+        requireNonNull(person);
+        return persons.containsPhone(person);
+    }
+
+    /**
+     * Returns true if a person with the same phone number as {@code person} exists in the address book.
```

Comment:
The comment is not updated, it should be `with the same email`

Reply:
Fixed.